docs/overview/index.md

```diff
@@ -0,0 +1,23 @@
+---
+id: overview_index
+title: "Contents"
+---
+
+ZIO Actors - a high-performance, purely-functional library for building, composing, and supervising typed actors backed by `ZIO`.
+Here's list of contents available:
+
+ - **[Basics](basics.md)** — Instantiating `ActorSystem`, defining actor's behavior, spawning actors.
+ - **[Supervision](supervision.md)** — Short description of supervision functionality usage
+ - **[Remoting](remoting.md)** — Defining remoting configuration, usage example, restrictions
+ - **[Queues](queue.md)** — **Not implemented** Support for different types of queues within a mailbox
```

Comment:
I would remove the *not implemented* stuff, and make sure they're reflected in the appropriate issues. Keep in mind the premise - "contents available".

Reply:
Sure, removed. As I checked they're covered in issues.